samples/drivers/display/boards/longan_nano.conf

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SPI=y
```

Comment:
NACK for unnecessary overlay, canbe part of the board configuration, see https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/pinetime_devkit0/Kconfig.defconfig#L14 as example.